docs/index.md

```diff
@@ -15,1621 +15,118 @@ parent="smn_workw_machine"
 > don't recommend you use it in production yet.
 
 Machine lets you create Docker hosts on your computer, on cloud providers, and
-inside your own data center. It creates servers, installs Docker on them, then
-configures the Docker client to talk to them.
+inside your own data center. It automatically creates hosts, installs Docker on
+them, then configures the `docker` client to talk to them.  A "machine" is the
+combination of a Docker Host and a configured client.
```

Comment:
I think the casing of "Docker Host" implies it's some sort of special entity or concept, we should probably always case it as "Docker host".